syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -498,12 +498,8 @@ def sigmoid(self, method="exp"):
         """
 
         if method == "exp":
-            # Inverse can only be used on matrices
-            if len(self.shape) == 1:
-                one = self * 0 + 1
-                result = one / (1 + (self * -1).exp())
-            else:
-                result = (1 + (self * -1).exp()).inverse()
+            one = self * 0 + 1
+            result = one / (1 + (self * -1).exp())
```

Comment:
So basically you're suggesting .inverse() doesn't work ? :(
(The benefit of .inverse is that it's supposed to be faster)

Reply:
But yeah you're right it's failiing apparently sometimes, I was discussing of this two weeks ago on the call with André